istioctl/pkg/writer/envoy/clusters/clusters.go

```diff
@@ -25,6 +25,7 @@ import (
 
 	adminapi "github.com/envoyproxy/go-control-plane/envoy/admin/v3"
 	core "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
+	"sigs.k8s.io/yaml"
```

Comment:
Can you use `github.com/ghodss/yaml` package for  `yaml.JSONToYAML`. It will be consistent across other codebase.

Reply:
sigs.k8s.io/yaml is also used in some places across istio repos and sigs.k8s.io/yaml is more active. Shouldn't we move to sigs.k8s.io/yaml?

Reply:
`sigs.k8s.io/yaml` is a fork of `ghodss/yaml` so it won't hurt. There was some discussion late last year to stick to one yaml lib but I think as `sigs.k8s.io/yaml` well maintained and active, we can stick to it.